pages/Diet-Cookie.md

```diff
@@ -0,0 +1,3 @@
+*"A very lightweight cookie."*
```

Comment:
There is no need to repeat the lore on here, it doesn't really give much information about the Item.